app.js

```diff
@@ -168,6 +173,13 @@ d.run(function () {
 			var cors = require('cors');
 			var app = express();
 
+			if (appConfig.coverage) {
+				var im = require('istanbul-middleware');
+				logger.debug('Hook loader for coverage - ensure this is not production!');
```

Comment:
running coverage on production will just gather code coverage on production, which just adds some extra work for the nodes.